programs/eosioc/main.cpp

```diff
@@ -1006,7 +1008,7 @@ int main( int argc, char** argv ) {
          }
       } else {
          // attempt to extract the error code if one is present
-         if (verbose_errors || !print_help_text(e)) {
+         if (verbose_errors || !print_help_text(e, show_error_trace)) {
             elog("Failed with error: ${e}", ("e", verbose_errors ? e.to_detail_string() : e.to_string()));
```

Comment:
Is there any downside to unifying `verbose_errors` and `show_error_trace`?  It seems like in cases where the help text is present, `verbose_errors` just forces the old style ugly printing.  Semantically, "verbose" to me could imply traces.

Reply:
Actually... you are right. I don't really pay attention on what "verbose_errors" had been printing until now and now I notice that the information is redundant with the updated print_help_text function.  
Thanks for pointing out!

PR Updated for this purpose, also I modify print_help_function to print more information when it is verbose.

Now it looks like this
![screen shot 2018-02-27 at 6 28 09 pm](https://user-images.githubusercontent.com/6541441/36724260-9fbb4210-1bed-11e8-9323-f0b64e8da496.png)